src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -2380,3 +2383,19 @@ class FunctionSegment(ansi.FunctionSegment):
             ),
         ),
     )
+
+
+class FromClauseSegment(ansi.FromClauseSegment):
+    """Slightly modified version which allows for using brackets for content of FROM."""
+
+    match_grammar: Matchable = StartsWith(
+        "FROM",
+        terminator=Ref("FromClauseTerminatorGrammar"),
+        enforce_whitespace_preceding_terminator=True,
+    )
+    parse_grammar: Optional[Matchable] = Sequence(
```

Comment:
Most of the time in this file (though I admit the code is inconsistent), we don't use the type hints, so I think it could be: `parse_grammar = Sequence(` and we don't need to `from typing import Optional`

Reply:
agreed. `parse_grammar` already has a type hint in `BaseSegment` - so no need to retype it here.